### PR TITLE
Fix Swagger base path

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #725, #722
| License       | MIT
| Doc PR        | n/a

Alternative fix to #725. It is done the same way than in the Hydra doc.

ping @Ener-Getick

### CODE REVIEW

**Comment by Ener-Getick** [comment]
LGTM :+1: it avoids introducing a new interface even if instantiating objects is more and more complicated outside of the framework :/

**Comment by Simperfit** [comment]
LGTM

File: src/Swagger/Serializer/DocumentationNormalizer.php [hunk 1]
```diff
@@ -455,20 +458,28 @@ private function getPropertySchema(PropertyMetadata $propertyMetadata) : \ArrayO
...
+    private function computeDoc(Documentation $documentation, \ArrayObject $definitions, \ArrayObject $paths) : array
     {
         $doc = [
             'swagger' => self::SWAGGER_VERSION,
-            'info' => ['title' => $object->getTitle()],
+            'basePath' => $this->urlGenerator->generate('api_entrypoint'),
+            'info' => [
+                'title' => $documentation->getTitle(),
+                'version' => $documentation->getVersion(),
+                'description' => $documentation->getDescription(),
```
Comment (teohhanhui) [inline src/Swagger/Serializer/DocumentationNormalizer.php]: The default value of the description is the empty string, and we don't want to use that, do we?

Comment (dunglas) [inline src/Swagger/Serializer/DocumentationNormalizer.php]: The same apply for the title and the version in this case.

Comment (Ener-Getick) [inline src/Swagger/Serializer/DocumentationNormalizer.php]: No, there are required by the spec http://swagger.io/specification/#infoObject.

Comment (dunglas) [inline src/Swagger/Serializer/DocumentationNormalizer.php]: Fixed

**Pull request merged**